docs/built-in-bpa-rules.md

```diff
@@ -191,4 +191,46 @@ Azure supports two deployment models: Classic and Azure Resource Manager (ARM).
 Azure supports two deployment models: Classic and Azure Resource Manager (ARM). ARM provides several security enhancements to the Classic model. You can learn more about the differences [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/deployment-models).
 
 **Recommendation**: Use ARM for your storage accounts to provide security enhancements such as: stronger access control (RBAC), better auditing, ARM-based deployment and governance, access to managed identities, access to key vault for secrets, Azure AD-based authentication and support for tags and resource groups for easier security management.
-#### Severity: 1
\ No newline at end of file
+#### Severity: 1
+
+### TA-000032: API Management calls to API backends should not bypass certificate thumbprint or name validation
+API Management calls to API backends should not bypass certificate thumbprint or name validation.
+
+**Recommendation**: To improve security, API Management should validate the backend server certificate for all API calls. Enable SSL certificate thumbprint and name validation by setting the *tls.validateCertificateName* and *tls.validateCertificateChain* properties to true
+#### Severity: 2
+
+### TA-000033: API Management direct management endpoint should not be enabled
+The direct management REST API in Azure API Management bypasses Azure Resource Manager role-based access control, authorization, and throttling mechanisms, thus increasing the vulnerability of your service.
+
+**Recommendation**: Set the *properties.enabled* property to false, to ensure direct access to the management REST API is disabled
+#### Severity: 3
+
+### TA-000034: API Management minimum API version should be set to 2019-12-01 or higher
+To prevent service secrets from being shared with read-only users, the minimum API version should be set to 2019-12-01 or higher.
+
+**Recommendation**: Set the *apiVersionContstraint.minAPIversion* property to 2019-12-01 or higher to prevent service secrets from being shared with read-only users.
+#### Severity: 2
+
+### TA-000035: API Management Named Value secrets should be stored in Azure Key Vault
+Named values are a collection of name and value pairs in each API Management service. Secret values can be stored either as encrypted text in API Management (custom secrets) or by referencing secrets in Azure Key Vault. To improve security of API Management and secrets, reference secret named values from Azure Key Vault. Azure Key Vault supports granular access management and secret rotation policies.
+
+**Recommendation**: To utilize secrets stored in Key Vault for Azure API Management, learn more [here](https://learn.microsoft.com/en-us/azure/api-management/api-management-howto-properties?tabs=azure-portal#key-vault-secrets).
```

Comment:
```suggestion
**Recommendation**: To utilize secrets stored in Key Vault for Azure API Management, learn more [here](https://learn.microsoft.com/azure/api-management/api-management-howto-properties?tabs=azure-portal#key-vault-secrets).
```

Reply:
Thanks updated in both docs & src/Analyzer.Core/Rules/BuiltInRules.json